src/Faker/Provider/et_EE/Person.php

```diff
@@ -0,0 +1,251 @@
+<?php
+
+namespace Faker\Provider\et_EE;
+
+class Person extends \Faker\Provider\Person
+{
+    protected static $titleMale = array('Härra', 'Hr.');
+
+    protected static $titleFemale = array('Proua', 'Pr.', 'neiu', 'preili');
+
+    /**
+     * @link http://www.eki.ee/books/ekk09/index.php?p=6&p1=5&id=548
```

Comment:
copyright is of the book, so if I was quoting a book, it would be copyrighted. 

peoples' names cannot be copyrighted.